ast/term.go

```diff
@@ -1792,7 +1792,8 @@ type Object interface {
 	MergeWith(other Object, conflictResolver func(v1, v2 *Term) (*Term, bool)) (Object, bool)
 	Filter(filter Object) (Object, error)
 	Keys() []*Term
-	Elem(i int) (*Term, *Term)
+	//Elem(i int) (*Term, *Term)
```

Comment:
Let's more, not comment out 🙃

Reply:
It's removed now. If we really ought to keep it, LMK.